SUMMARY.md

```diff
@@ -11,21 +11,15 @@
 
 ## Wallets
 
-* [Arweave Web Extension Wallet](wallets/arweave-web-extension-wallet.md)
+* [Arweave.app Web Wallet](wallets/arweave-web-extension-wallet.md)
```

Comment:
suggestion: rename this file, otherwise the URL will be for the Arweave web extension

Reply:
I thought about it, I was worried about breaking existing bookmarks and references....

Reply:
create a new page and a redirect from the old one? 
pretty standard to have an "update your bookmarks... ...you will be redirected in 5 seconds" page

Reply:
....hold that thought